app/person.rb

```diff
@@ -1,8 +1,9 @@
-class Person
+class Person < Nameable
   attr_accessor :age, :name, :parent_permission
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
+    super(name)
```

Comment:
- [ ] The `super()` method in the `Person` constructor does not require any argument, as per the parent `Nameable` class, hence your app breaks. Kindly ensure to improve upon the same:

![image](https://user-images.githubusercontent.com/12745474/208879519-a12d5b0c-e2b5-41bf-9a6f-d41ba3d35cc7.png)

Reply:
Oh! I was just trying something out and forgot to remove it!